src/modules/vendor/components/questionnaires/ProfileEdition.vue

```diff
@@ -1,13 +1,22 @@
 <template>
   <div class="column">
-    <div class="row">
-      <ni-input v-model.trim="questionnaire.title" required-field caption="Titre" @blur="updateQuestionnaire"
-        @focus="saveTmpTitle" :error="$v.questionnaire.title.$error" :disable="questionnaire.status === PUBLISHED" />
+    <div class="row justify-between">
+      <div class="row body">
+        <ni-input v-model.trim="questionnaire.title" required-field caption="Titre" @blur="updateQuestionnaire"
+          @focus="saveTmpTitle" :error="$v.questionnaire.title.$error" :disable="titleLock" />
+        <ni-button v-if="isQuestionnairePublished" color="black" :icon="lockIcon"
+          @click.native="titleLock = !titleLock" />
```

Comment:
a quoi sert le `native` ?

Reply:
en effet je n'ai pas l'impression qu'il soit utile ici, je l'avais pris de profileinfo pour les emails. Et je suis pas sûr qu'il soit utile non plus de ce côté. Je l'enlève ?

Reply:
yes !

Reply:
done

Reply:
Sur la doc vue, ils disent que .native ne sert que lorsqu'on veut remplacer une fonction native (ici @click) sur un composant non natif.
https://dev.to/yossiabramov/the-native-event-modifier-in-vue-js-bpb

D'où le fait qu'on en ai pas besoin sur ni-button dans ce cas, parce que ni-button défini sa fonction @click.
Donc la fonction @click dans les props ne renvoi pas vers la fonction native click mais vers une méthode du composant ni-button (comme si on appelait `handleClickOnTheButton()` sauf que les deux fonctions, la native et la method de ni-button, s'appellent pareil).


Mais du coup je ne comprends pas pourquoi on a besoin de .native sur le q-button du ni-button ? Les composant quasar ne sont pas considéré comme des composants natifs par vue ?

Reply:
c'est possible que non, de même qu'un composant expo n'est pas un composant react native mais un composant... expo, non ?